src/thread.rs

```diff
@@ -226,7 +229,7 @@ pub struct ScopedJoinHandle<'a, T: 'a> {
 /// ```
 /// crossbeam_utils::thread::scope(|scope| {
 ///     scope.defer(|| println!("Exiting scope"));
-///     scope.spawn(|| println!("Running child thread in scope"))
+///     scope.spawn(|| println!("Running child thread in scope"));
```

Comment:
Without the semicolon the `ScopedJoinHandle` was actually returned by the invocation of `scope()`, thus leaking out of the scope!